config/profile_stub.config

```
@@ -12,6 +12,7 @@ params{
   cellhash_pool_file = 'test/stub-pools.tsv'
   outdir = "test/output"
   run_ids = "All"
+  celltype_project_metafile = 'test/celltypes/stub-celltype-project-metadata.tsv'
```

Moving started just for organization, but also suggesting a slightly different location for this file. (Which we may want to move in the admin folder as well. I'm not sure I understand the current organization of the celltyping files there... I'm not sure why the reference files/scripts are nested in `sample_info`; seems like they should be in a different place and possibly not in `scpca-admin` at all.)
```suggestion
  celltype_project_metafile = 'test/stub-celltype-project-metadata.tsv'
  outdir = 'test/output'
  run_ids = 'All'
```